packages/amazonq/.vscode/launch.json

```diff
@@ -18,7 +18,8 @@
             "args": ["--extensionDevelopmentPath=${workspaceFolder}"],
             "env": {
                 "SSMDOCUMENT_LANGUAGESERVER_PORT": "6010",
-                "WEBPACK_DEVELOPER_SERVER": "http://localhost:8080"
+                "WEBPACK_DEVELOPER_SERVER": "http://localhost:8080",
+                "AWS_VSCODE_DEBUG_MODE": "true"
```

Comment:
Alternatively can't we just check that the WEBPACK_DEVELOPER_SERVER is defined to tell its in dev mode? Also is there a launch config for the toolkit as well that makes these changes?

Reply:
That makes sense for sure. Will update.

Yeah there is one already, if you see the individual commit it is easier to review